hokusai/production.yml

```diff
@@ -80,6 +80,8 @@ apiVersion: networking.k8s.io/v1beta1
 kind: Ingress
 metadata:
   name: aprd
+  annotations:
+    nginx.ingress.kubernetes.io/whitelist-source-range: "{{ cloudflareIpSourceRanges|join(',') }}"
```

Comment:
I think this is supposed to be `externalIngressAllowSourceIP` after the discussion in https://github.com/artsy/waves/pull/293.

Reply:
Ah right I forgot the outcome of that discussion.  Will update.